### PR TITLE
added CCLE SNP Fingerprinting pipeline

### PR DESCRIPTION
**Pull request by colganwi**
Pipeline updates the LOD matrix on Taiga. We will need to add downstream analysis to this once we determine what analyses are most useful.

### CODE REVIEW

File: RNA_CCLE.ipynb [hunk 1]
```diff
@@ -1434,7 +1434,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
```
Comment (jkobject) [inline RNA_CCLE.ipynb:1435]: can you remove this change from the pull request?

File: Fingerprint_pipeline/make_ccle_29k_coding_snps_step2.R [hunk 2]
```diff
@@ -0,0 +1,46 @@
+library(tidyverse)
+# These steps are required for lifting over the map to hg38
```
Comment (jkobject) [inline Fingerprint_pipeline/make_ccle_29k_coding_snps_step2.R:2]: could you push a python version of this instead?

How should I call it? what are the inputs? what are the outputs?

File: Fingerprint_pipeline/make_ccle_29k_coding_snps_step1.sh [hunk 3]
```diff
@@ -0,0 +1,22 @@
+#!/bin/bash -l
+# Download and install https://github.com/naumanjaved/fingerprint_maps
+# Then run this bash script for all chromosomes except for Y
+CHR="22"
```
Comment (jkobject) [inline Fingerprint_pipeline/make_ccle_29k_coding_snps_step1.sh:4]: this bash script wont work as is.

maybe can add a for loop across all chromosome, 
also maybe add mkdirs for missing directoriess

Comment (jkobject) [reply Fingerprint_pipeline/make_ccle_29k_coding_snps_step1.sh:4]: do I need GATK installed? do I need a specific version?

Comment (jkobject) [reply Fingerprint_pipeline/make_ccle_29k_coding_snps_step1.sh:4]: also I need bcftools, wget, python (any version?)
also this file is not here I think: ~/Data/Fingerprinting/fingerprint_maps/build_fingerprint_maps.py

File: Fingerprint_pipeline/make_ccle_29k_coding_snps_step2.R (hunk 2 above)
Comment (jkobject) [reply Fingerprint_pipeline/make_ccle_29k_coding_snps_step2.R:2]: can we push this multi step pipeline into one part?

**Review by jkobject — CHANGES_REQUESTED** [review]
I have a couple questions listed in comments.
Also I would advice keeping the data (map files) in the ccle buckets with the rest of the ccle data.

Also I have seen this multi step pipeline to make the snps. Do you plan on updating it later? 
If this is something I might handle in the future, I have a couple questions / requests.

File: Fingerprint_pipeline/conf_fingerprint_bam_with_liftover.json [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+  'fingerprint.run_fingerprint.contamination': '',
+  'fingerprint.run_fingerprint.bam_index': 'this.bai_filepath',
+  'fingerprint.run_fingerprint.liftover_ref_fasta': 'workspace.referenceData_hg38_ref_fasta',
+  'fingerprint.run_fingerprint.output_name': 'this.sample_id',
+  'fingerprint.run_fingerprint.bam': 'this.bam_filepath',
+  'fingerprint.run_fingerprint.bam_ref_fasta_index': 'workspace.referenceData_b37Human_ref_fasta_index',
+  'fingerprint.run_fingerprint.hardware_disk_size_GB': '',
+  'fingerprint.run_fingerprint.liftover_ref_fasta_index': 'workspace.referenceData_hg38_ref_fasta_index',
+  'fingerprint.run_fingerprint.liftover_ref_dict': 'workspace.referenceData_hg38_ref_dict',
+  'fingerprint.run_fingerprint.liftover_chain': '"gs://fc-secure-6b6a3e1a-6fb8-4d30-b0df-a359e6c5d6e6/b37ToHg38.over.chain"',
```
[the remark this answers is not in the document]
Comment (colganwi) [reply Fingerprint_pipeline/conf_fingerprint_bam_with_liftover.json:13]: Sounds good. I'll switch the paths.

File: Fingerprint_pipeline/make_ccle_29k_coding_snps_step1.sh (hunk 3 above)
Comment (jkobject) [reply Fingerprint_pipeline/make_ccle_29k_coding_snps_step1.sh:4]: ok let's put it in a folder: Fingerprintin../tasks/ to make differentiate it from non One-Off tasks.

File: Fingerprint_pipeline/fingerprint_bam.wdl [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+  	java "-Xmx${hardware_memory_GB}g" -jar /usr/picard/picard.jar ExtractFingerprint \
+      --INPUT "${bam}" \
+      --OUTPUT "${output_name}.vcf" \
+      --HAPLOTYPE_MAP "${haplotype_map}" \
+      --REFERENCE_SEQUENCE "${ref_fasta}" \
+      ${"--CONTAMINATION " + contamination} \
+      ${"--SAMPLE_ALIAS " + sample_alias} \
+      ${"--LOCUS_MAX_READS " + locus_max_reads}
+    
+    gsutil cp "${output_name}.vcf" "${output_dir}/${output_name}.vcf"
```
Comment (jkobject) [inline Fingerprint_pipeline/fingerprint_bam.wdl:33]: hey, you still have this step that is not necessary, except if I am missing something

**Review by jkobject — APPROVED** [review]
there is one line that I don't understand otherwise you can merge the pull request! :)

**Pull request merged**